File: android/src/main/java/com/github/mikephil/charting/highlight/IDataSetHighlighter.java
package com.github.mikephil.charting.highlight;

import com.github.mikephil.charting.highlight.DataSetHighlight;

public interface IDataSetHighlighter {

    /**
     * Returns the index corresponding to the highlighted DataSet near the given x- and y- touch positions in pixels.
     *
     * @param x
     * @param y
     * @return
     */
    DataSetHighlight getDataSetIndexHighlight(float x, float y);
}

File: ios/Charts/Highlight/IDataSetHighlighter.swift
import Foundation
import CoreGraphics

@objc(IChartDataSetHighlighter)
public protocol IDataSetHighlighter: class
{
    /// - Parameters:
    ///   - x:
    ///   - y:
    /// - Returns the index corresponding to the highlighted DataSet near the given x- and y- touch positions in pixels.
    func getDataSetIndexHighlight(x: CGFloat, y: CGFloat) -> Int
}
